docs/reference-range.rst

```diff
@@ -15,7 +15,7 @@ Data model
 .. csv-table::
    :header: Field, Type, Multiplicity, Description
 
-   unit, :ref:`rstontologyclass`, required, Ontology term describing the unit.
```

Comment:
Needs `REQUIRED.` adding after the description, like the `high` and `low` fields.